data/soapvalidator/MailClient/Calendar/Bugs/bug62674.xml

```diff
@@ -61,7 +71,7 @@
 
 	<t:test >
         <t:request>
-            <CreateGalSyncAccountRequest xmlns="urn:zimbraAdmin" name="name${TIME}${COUNTER}" type="zimbra" domain="${domain1.name}" server="${server.zimbraAdmin}">
+            <CreateGalSyncAccountRequest xmlns="urn:zimbraAdmin" name="name${TIME}${COUNTER}" type="zimbra" domain="${domain1.name}" server="${mailbox.server}">
```

Comment:
Please update commit comment to explain why this server change is made.

Reply:
server.zimbraAdmin is returning the proxy server. 
For CreateGalSyncAccountRequest to work, we need to specify the mail-host on which this GalSync account resides.